Makefile

```diff
@@ -0,0 +1,15 @@
+docker_image = resinos
+
+all: build_docker_image run_test
+
+build_docker_image: Dockerfile
+	@echo '[Info] Building docker image "$(docker_image)"...'
+	docker build -t $(docker_image) .
+
+run_test: env.list
+	@echo '[Info] Starting tests inside container...'
+	docker run -it --rm --env-file ./env.list $(docker_image) npm start
+
+clean:
```

Comment:
All these phony targets should be marked with `.PHONY` (https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html)